.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "nuget" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "KinsonDigital/CASL"
```

Add a single reviewer as shown below:

```suggestion
    assignees:
      - "CalvinWilkinson"
    reviewers:
      - "KinsonDigitalAdmin"
```